interfaces/apparmor/template.go

```diff
@@ -362,3 +362,35 @@ var defaultTemplate = []byte(`
 ###SNIPPETS###
 }
 `)
+
+// classicTemplate contains apparmor template used for snaps with classic confinement.
+//
+// It can be overridden for testing using MockClassicTemplate().
+// Designed by jdstrand https://github.com/snapcore/snapd/pull/2366#discussion_r90101320
+var classicTemplate = []byte(`
+#include <tunables/global>
+
+###VAR###
+
+###PROFILEATTACH### (attach_disconnected) {
```

Comment:
Sure. I'm more concerned about the user experience here. ',complain' or not, the profile is wide open so we shouldn't support --jailmode with --classic at this time since that will convey to the user that the --classic snap is sandboxed in some way. Also, --devmode doesn't make sense either, because there won't be any violations against policy and so there won't be anything in the logs showing what needs to be adjusted.

As such, either don't support --devmode/--jailmode with --classic (and just put ',complain' here as recommended for future proofing), or use the default template with --jailmode and --devmode (that is probably an exceedingly poor experience though-- the snap developer should just use 'confinement: strict' in that case).